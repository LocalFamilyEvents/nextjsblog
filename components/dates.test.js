import renderer from 'react-test-renderer';

import Date from "./date";


test("render correctly", () => {
    const component = renderer.create(
        <Date dateString='2022-02-02' />,
    );
    
    let json = component.toJSON();
    expect(json).toMatchSnapshot();
    // expect().toBe(<time dateTime='2022-02-02'>2nd Feb 2022</time>);

});
