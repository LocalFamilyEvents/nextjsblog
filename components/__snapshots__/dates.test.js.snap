// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<time
  dateTime="2022-02-02"
>
  February 2, 2022
</time>
`;
